test/e2e/managed_cluster_validating_webhooks_test.go

```diff
@@ -201,20 +199,17 @@ var _ = Describe("Managed Cluster Validating Webhooks", Ordered, func() {
 			}
 		})
 
-		withNamespace := func(pod *v1.Pod, namespace string) *v1.Pod {
-			pod.SetNamespace(namespace)
-			return pod
-		}
```

Comment:
This can be removed and the code simplified by setting the `privilegedNamespace` as the default `ns` in the pod's `ObjectMeta` definition above.